src/Domain/Entity/Service.php

```diff
@@ -175,17 +174,15 @@ public function isTv(): bool
     public function isRadio(): bool
     {
         // Network can be null, but unfetched should throw an exception
-        if ($this->network && $this->getNetwork()->isRadio()) {
-            return true;
-        }
-        return false;
+        $network = $this->getNetwork();
+
+        return $network && $$network->isRadio();
```

Comment:
$$network?

Reply:
Hmm. Looking at this (aside from the typo), this will throw an exception if network is unfetched, where the old code would have simply returned false. Is that what we want?

Reply:
The old code would also have thrown the exception. Wouldn't it?

Reply:
I think we should be clear and shouty when we try and access some information that we haven't retrieved yet to avoid those insidious "why is everything powering through and returning false" bugs.